lib/iris/tests/test_merge.py

```diff
@@ -685,11 +683,9 @@ def test_simple3(self):
                                    ])
         cube = cubes.merge()[0]
         self.assertCML(cube, ('merge', 'time_triple_merging4.cml'), checksum=False)
-        self.assertIsNone(cube.assert_valid())
 
         cube = iris.cube.CubeList(cubes[:-1]).merge()[0]
         self.assertCML(cube, ('merge', 'time_triple_merging5.cml'), checksum=False)
-        self.assertIsNone(cube.assert_valid())
```

Comment:
I can't work out what these tests were trying to do. This code has been here since the "intial commit"